video/beta/src/main/java/com/example/video/TrackObjects.java

```diff
@@ -0,0 +1,171 @@
+import com.google.cloud.videointelligence.v1p2beta1.NormalizedBoundingBox;
+import com.google.cloud.videointelligence.v1p2beta1.ObjectTrackingAnnotation;
+import com.google.cloud.videointelligence.v1p2beta1.ObjectTrackingFrame;
+import com.google.cloud.videointelligence.v1p2beta1.VideoAnnotationResults;
+import com.google.cloud.videointelligence.v1p2beta1.VideoIntelligenceServiceClient;
+import com.google.cloud.videointelligence.v1p2beta1.VideoSegment;
+import com.google.protobuf.ByteString;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+public class TrackObjects {
+
+  // [START video_object_tracking_beta]
+  /**
+   * Track objects in a video.
+   *
+   * @param filePath the path to the video file to analyze.
+   */
+  public static VideoAnnotationResults trackObjects(String filePath) throws Exception {
+
+    try (VideoIntelligenceServiceClient client = VideoIntelligenceServiceClient.create()) {
+
+      // Read file
+      Path path = Paths.get(filePath);
+      byte[] data = Files.readAllBytes(path);
+
+      // Create the request
+      AnnotateVideoRequest request = AnnotateVideoRequest.newBuilder()
+          .setInputContent(ByteString.copyFrom(data))
+          .addFeatures(Feature.OBJECT_TRACKING)
+          .setLocationId("us-east1")
+          .build();
+
+      // asynchronously perform object tracking on videos
+      OperationFuture<AnnotateVideoResponse, AnnotateVideoProgress> response =
+          client.annotateVideoAsync(request);
+
+      System.out.println("Waiting for operation to complete...");
+      // The first result is retrieved because a single video was processed.
+      VideoAnnotationResults result = response.get(1000, TimeUnit.SECONDS)
+          .getAnnotationResults(0);
+
+      // Get only the first annotation for demo purposes.
+      ObjectTrackingAnnotation objectTrackingAnnotation = result.getObjectAnnotations(0);
+
+      if (objectTrackingAnnotation.hasEntity()) {
+        System.out.println("Entity description: "
+            + objectTrackingAnnotation.getEntity().getDescription());
+        System.out.println("Entity id:: " + objectTrackingAnnotation.getEntity().getEntityId());
+      }
+
+      if (objectTrackingAnnotation.hasSegment()) {
+        VideoSegment videoSegment = objectTrackingAnnotation.getSegment();
+        System.out.println(String.format("Segment: %.2fs to %.2fs",
+            videoSegment.getStartTimeOffset().getSeconds()
+                + videoSegment.getStartTimeOffset().getNanos() / 1e9,
+            videoSegment.getEndTimeOffset().getSeconds()
+                + videoSegment.getEndTimeOffset().getNanos() / 1e9));
+      }
+
+      System.out.println("Confidence: " + objectTrackingAnnotation.getConfidence());
+
+
+      // Here we print only the bounding box of the first frame in this segment.
+      ObjectTrackingFrame objectTrackingFrame = objectTrackingAnnotation.getFrames(0);
```

Comment:
s/objectTrackingFrame/frame

Unless it leads to conflicting variable names, dropping adjectives from long class names helps make the code easier to read.